### ISSUE TITLE
Simplifying observation/photo/sound licenses

### ISSUE THREAD

**Issue by loarie** [inaturalist/inaturalist#637]
When someone joins iNaturalist, they get separate CC-BY-NC licenses added to their observations, photos and sounds unless they un-check the 'Yes license my...' box.

Not surprisingly, the vast majority of iNat users (78%) go with these settings (the NC-NC pink block in the lower left of the graphs). There's another cohort of around 5% who uncheck the box (the ARR-ARR pink block towards the center of the graphs).

The interesting cohort, is the cohort that modifies these default bundled settings. I thought the remaining pink blocks on this graph represented large cohorts that actively changed their settings, but unfortunately I think these are mostly artifacts for how we've rolled things out over the years (e.g. switching from CC-BY to CC-BY-NC, rolling out licenses first, and then sound) and how people opted in or out of these features as they were introduced, so I'm not sure there's a lot to learn from the data without very rigorously retracing exactly how we rolled things out... oh well.
<img width="1427" alt="screen shot 2015-07-16 at 4 20 40 pm" src="https://cloud.githubusercontent.com/assets/598026/8737426/b7b3fc36-2bd6-11e5-8434-6d7db9277479.png">
But it does seem that most users aren't doing much other than leaving the settings checked, unchecking them, or perhaps actively altering things in a way that makes photos and/or sounds AllRightsReserved but not Observations (if these aren't artifacts). But its clear that people are NOT opting out of Observations licensing independently of the 'bundled opt out' which is almost certainly driven by concern over media (not observations).

With that in mind, here's a proposal to:
(a) make it so that observations must be licensed for new users
(b) combine photo and sound licenses for new users
in order to greatly simplify what it means for an observation to be 'licensed'. With these changes, 'licensed observations' will have CC licenses applied to their media (photos/sound). 'unlicensed observations' will have media that is AllRightsReserved.

In the iNat terms of service we can make it clear that observations created on iNat are considered licensed content without explicitly displaying licenses for the non-media portion of an observation.

**Comment by tnywlls** [comment]
For what it's worth, one can't really copyright 'an observation', in the sense of copyrighting the fact that you observed or reported observing something, only copyrightable element (other than media) is the creative content in the writing of a description and/or comments if any.

**Comment by kueda** [comment]
@tnywlls, that's correct in US copyright law, but when we originally introduced licensing we thought it was already confusing enough, and trying to convey the fact that only an observation with a description or tags could even be considered copyrightable in the US was too complex. Also, we are an international website, and IP laws concerning databases vary quite a bit across national jurisdictions, so your statement is not necessarily correct elsewhere (the US is actually kind of progressive in this regard). http://opendefinition.org/guide/data/ provides a good overview. To be really solid about licensing, we should probably consider a different, more data-oriented license for the observation

Scott, are you proposing that we remove the ability to declare a license for your observations, or just remove that option from the signup process?

**Comment by tnywlls** [comment]
Thanks for the link, a good summary.  My point is that I wonder what someone expects when they copyright an observation on iNat.  Do they really care whether you copy their text, or are they thinking they are protecting the information that they saw xyz at abc on nnnn:nn:nn.   Or do they think they are invoking a database copyright for all their observations?
I think the cleanest, and most useful approach is to just make it a condition of use of the site that any observation data is openly licensed for all to use, but they may restrict usage of media.

**Comment by carrieseltzer** [comment]
I feel like I've noticed more people recently who have "all rights
reserved" on their observations (and media) and they seem to:
1. be using a mobile app.
2. often have numbers in their username.
3. have their full name on their profile.

Here are a couple of examples from today:
http://www.inaturalist.org/observations/2346107
http://www.inaturalist.org/observations/2346706
http://www.inaturalist.org/observations/2346315

This leads me to suspect that the default setting for creating a new
account from a mobile app via Facebook (or google) is ARR rather than CC
BY-NC. Can someone confirm? Does creating an account via FB or G+ apply ARR
settings by default? Happy to start a new thread for this if appropriate.

Couldn't test this myself as my Facebook account is already linked to my
iNat account.


On Thu, Oct 15, 2015 at 6:54 AM, tnywlls <notifications@github.com> wrote:

> Thanks for the link, a good summary. My point is that I wonder what
> someone expects when they copyright an observation on iNat. Do they really
> care whether you copy their text, or are they thinking they are protecting
> the information that they saw xyz at abc on nnnn:nn:nn. Or do they think
> they are invoking a database copyright for all their observations?
> I think the cleanest, and most useful approach is to just make it a
> condition of use of the site that any observation data is openly licensed
> for all to use, but they may restrict usage of media.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inaturalist/inaturalist/issues/637#issuecomment-148350457>
> .
>



-- 
Carrie E. Seltzer, Ph.D.
National Geographic's Great Nature Project
www.greatnatureproject.org

**Comment by loarie** [comment]
Good sleuthing Carrie, I made an issue https://github.com/inaturalist/design/issues/49

**Comment by loarie** [comment]
Once we implement this, I propose we get rid of the bottom right section of Obs/show now dedicated to observation licensing and flagging inappropriate obs by doing the following:
1) move licensing edit link to the photo license section below the photo (black arrow)
2) add 'flag' link to Appropriate section in Data Quality Assessment (red arrow)
3) add language about what inappropriate means to modal for flagging 
(green arrow)
<img width="581" alt="screen shot 2015-11-04 at 11 09 33 pm" src="https://cloud.githubusercontent.com/assets/598026/10962002/29b5b774-8349-11e5-8d9a-09804f8b7a75.png">
Also, looks like some weird formatting in the Appropriate section currently when obs are flagged:
<img width="274" alt="screen shot 2015-11-04 at 11 13 47 pm" src="https://cloud.githubusercontent.com/assets/598026/10962055/b90e4ed6-8349-11e5-8d55-3db479adc3ce.png">

**Comment by loarie** [comment]
>>Scott, are you proposing that we remove the ability to declare a license for your observations, or just remove that option from the signup process?

@kueda sorry just saw this, I'm proposing we remove the ability to declare a license for observations. Specifically:

Step 1:
Wire up Signup so that:
Checking the 'license my content' box sets photo: CC-BY-NC, sound: CC-BY-NC, observation: CC-BY
Unchecking the box means photo: nil, sound: nil, observation: CC-BY

Step 2: 
Alter Account UI so that:
1. For new users, instead of 3 choosers there's just a single 'Default Media License' chooser, e.g. like this but labeled differently:
<img width="388" alt="screen shot 2015-11-19 at 4 12 17 pm" src="https://cloud.githubusercontent.com/assets/598026/11288399/81e2c3d4-8ed8-11e5-9921-a023da95b6a5.png">
2. For old users add the following transition tool above the existing Licensing UI:
```
We've simplified how licenses work and now only support a single media license for photos AND sound rather then separate licenses for observations, sound and photos. For new users, all observations are automatically given a CC-BY license that can't be altered and a single Media license applies to both photos and sounds. As an existing user, nothing has changed for you, but we encourage you to adopt this simpler system. This will:
1) apply a CC-BY license to all of your past observations and set this as your license moving forward
2) apply your current photo license to all of your past sounds as a single media license
3) simplify this interface so that there is just a single media license chooser
(Yes please) (No thanks)
```
Where 'Yes Please' changes the user's content and switches to the new user License UI

Step 3:
Stop displaying observation licenses on observation/show for all observations:
<img width="311" alt="screen shot 2015-11-19 at 4 16 58 pm" src="https://cloud.githubusercontent.com/assets/598026/11288452/ff22f33c-8ed8-11e5-9f30-eafb843e8cf1.png">
